src/app/component/comments/components/comments-list/comments-list.component.scss

```diff
@@ -211,6 +211,18 @@ $main-font: 'Open Sans', sans-serif;
       max-height: 32px;
     }
   }
+
+  .comment-details {
+    flex-direction: column;
+
+    .comment-date-likes {
+      justify-content: left;
+
+      .comment-likes {
+        margin-left: 9px;
```

Comment:
I don't really see the point in such nesting of elements in Anglar, especially in the case of encapsulation )